CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/proximity/FR3DArranger.java

```diff
@@ -140,6 +139,7 @@ protected Object run() throws InterruptedException {
     }
 
     private void initialise(final int width, final int height, final int depth) {
+        double forceConstant;
```

Comment:
can these be defined at the point they're instantiated (possibly made final if it makes sense)?